modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -529,8 +559,25 @@ export default {
         this.modal.showModal = false;
       }
     },
+    async onCopy(e) {
+      if (this.isModified) {
```

Comment:
Since the "copy" option is on the "more" menu of the doc editor, we have to do something sensible if the user has already made changes. I followed the pattern from A2 here, saving the changes, but I only "save draft" as it's never good to force a publication without the user clicking that button.

Reply:
I disagree with saving the changes back to the original document .. the user chose to jump out of that document without saving/publishing, we shouldn't do anything cute. I can think of instances where i've prepped a duplicate from the original state. We should trust the user on this.

Reply:
I agree, let's not take the wheel and make assumptions about the users intentions. We don't do this automatically elsewhere and I don't think this is an excuse to introduce the pattern. We would need to take a wholistic approach (like Gmail), applying this 'draft is saved when user leaves the doc' to all situations, which I don't think we should crack open right now.

Reply:
We might choose to give them a confirm modal, like we do elsewhere, when the user tries to do this. Thats the established pattern so far.

Reply:
We could introduce a confirm in this event if we're really keen on the value